### PR TITLE
ModelBuilder to fetch local schema when no SchemaBuilder present.

### PR DESCRIPTION
**Pull request by makungaj1**
*Issue #, if available:*

*Description of changes:*
ModelBuilder to fetch local schema when no SchemaBuilder present

*Testing done:*
* Unit tests
* Integ tests
* Notebook test

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [ x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [ x] I certify that the changes I am introducing will be backward compatible, and I have discussed concerns about this, if any, with the Python SDK team
- [ x] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [x ] I have passed the region in to all S3 and STS clients that I've initialized as part of this change.
- [ x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [ x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ x] I have added unit and/or integration tests as appropriate to ensure backward compatibility of the changes
- [x ] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [x] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-slow-tests
* Commit ID: e3361c2edc71c0322b4441e5ea2420cdfa487cf2
* Result: SUCCEEDED
* [Build Logs](https://omfrydptb9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=cfac7055-69dc-42a0-b709-033e83a99a9c%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-local-mode-tests
* Commit ID: e3361c2edc71c0322b4441e5ea2420cdfa487cf2
* Result: SUCCEEDED
* [Build Logs](https://8huhkqo601.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=b936ca36-65c7-44a0-a6c1-64b3e1b41de9%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-notebook-tests
* Commit ID: e3361c2edc71c0322b4441e5ea2420cdfa487cf2
* Result: SUCCEEDED
* [Build Logs](https://bkqkixfb59.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=c7c5378a-6624-4999-841d-e112c51708fa%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-unit-tests
* Commit ID: e3361c2edc71c0322b4441e5ea2420cdfa487cf2
* Result: SUCCEEDED
* [Build Logs](https://fegcnxu6da.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=4e5b5288-60b9-494e-b872-a2c1a65360e3%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-pr
* Commit ID: e3361c2edc71c0322b4441e5ea2420cdfa487cf2
* Result: FAILED
* [Build Logs](https://abkt2mlh3k.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=8bffbf30-477a-4d8c-8e42-9db890dcffcb%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: src/sagemaker/image_uri_config/tasks.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        "outputs": {
+			"ref": "https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/tasks/text-generation/spec/output.json",
+			"properties": [
+				{
+					"generated_text": "Hello, I'm a language modeler. So while writing this, when I went out to meet my wife or come home she told me that my"
+				}
+			]
+		}
+    }
+}
```
Comment (samruds) [inline src/sagemaker/image_uri_config/tasks.json:80]: Add space here

File: src/sagemaker/image_uri_config/tasks.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
+{
+	"description": "Sample Task Inputs and Outputs",
```
Comment (samruds) [inline src/sagemaker/image_uri_config/tasks.json:2]: Where is the description used?

File: src/sagemaker/serve/utils/exceptions.py [hunk 3]
```diff
@@ -60,3 +60,12 @@ class SkipTuningComboException(ModelBuilderException):
 
     def __init__(self, message):
         super().__init__(message=message)
+
+
+class TaskNotFoundException(ModelBuilderException):
+    """Raise when task could not be found"""
```
Comment (samruds) [inline src/sagemaker/serve/utils/exceptions.py:66]: When huggingface task

File: src/sagemaker/task.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+            if task_io_schemas is None:
+                raise ValueError(f"Could not find {task} I/O schema.")
+
+            sample_schema = (
+                task_io_schemas["inputs"]["properties"],
+                task_io_schemas["outputs"]["properties"],
+            )
+        return sample_schema
+    except FileNotFoundError:
+        raise ValueError("Could not find tasks config file.")
```
Comment (samruds) [inline src/sagemaker/task.py:49]: Can we add test coverage for this case?

File: src/sagemaker/serve/builder/model_builder.py [hunk 5]
```diff
@@ -672,3 +678,18 @@ def validate(self, model_dir: str) -> Type[bool]:
...
+            model_task (str): Required, the task name
+
+        Raises:
+            TaskNotFoundException: If the I/O schema for the given task is not found.
+        """
+        try:
+            sample_inputs, sample_outputs = task.retrieve_local_schemas(model_task)
+            self.schema_builder = SchemaBuilder(sample_inputs, sample_outputs)
+        except ValueError:
+            raise TaskNotFoundException(f"Schema builder for {model_task} could not be found.")
```
Comment (samruds) [inline src/sagemaker/serve/builder/model_builder.py:695]: Can we add test coverage for the exception cases?

File: src/sagemaker/task.py [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+# language governing permissions and limitations under the License.
+"""Accessors to retrieve task fallback input/output schema"""
+from __future__ import absolute_import
+
+import json
+import os
+from typing import Any, Tuple
+
+
+def retrieve_local_schemas(task: str) -> Tuple[Any, Any]:
```
Comment (samruds) [inline src/sagemaker/task.py:21]: I think this should be in serve/utils https://github.com/aws/sagemaker-python-sdk/tree/master/src/sagemaker/serve/utils

**Comment by gwang111** [comment]
When we apply these defaults, what are the max_token_lengths we are picking?

File: src/sagemaker/image_uri_config/tasks.json [hunk 7]
```diff
@@ -0,0 +1,80 @@
+{
```
Comment (jiapinw) [inline src/sagemaker/image_uri_config/tasks.json:1]: From the impl, 
```
task_io_schemas["inputs"]["properties"],
task_io_schemas["outputs"]["properties"],
```
we only use part of the config file. Are the rest necessary?

Comment (makungaj1) [reply src/sagemaker/image_uri_config/tasks.json:1]: No, had them there for desc. But will remove them to reduce the size of the file.

File: src/sagemaker/serve/utils/task.py [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+        task (str): Required, the task name
+
+    Returns:
+        Tuple[Any, Any]: A tuple that contains the sample input,
+        at index 0, and output schema, at index 1.
+
+    Raises:
+        ValueError: If no tasks config found or the task does not exist in the local config.
+    """
+    # task_io_config_path = os.path.join(os.path.dirname(__file__), "image_uri_config", "tasks.json")
```
Comment (samruds) [inline src/sagemaker/serve/utils/task.py:34]: Remove this comment?

Comment (makungaj1) [reply src/sagemaker/serve/utils/task.py]: done

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-local-mode-tests
* Commit ID: 99d11cbf51ce1c986c49c1151012ef8f5fdc81dc
* Result: FAILED
* [Build Logs](https://8huhkqo601.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=cce8d7e0-8051-4101-a0be-c815a5c774ef%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-notebook-tests
* Commit ID: 99d11cbf51ce1c986c49c1151012ef8f5fdc81dc
* Result: SUCCEEDED
* [Build Logs](https://bkqkixfb59.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=f040e905-5ed7-4af0-a63e-85cb85c84edb%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-unit-tests
* Commit ID: 99d11cbf51ce1c986c49c1151012ef8f5fdc81dc
* Result: SUCCEEDED
* [Build Logs](https://fegcnxu6da.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=2f9fdb01-a8b9-4cb4-9ed8-c940d60d4cb5%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by gwang111** [comment]
so `task.json` is sitting in the image_uri_config?

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-slow-tests
* Commit ID: 99d11cbf51ce1c986c49c1151012ef8f5fdc81dc
* Result: SUCCEEDED
* [Build Logs](https://omfrydptb9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=0239617b-5618-49d9-bda9-68247e446768%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by makungaj1** [comment]
> so `task.json` is sitting in the image_uri_config?

Yes, can double-check with @knikure

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-notebook-tests
* Commit ID: 38eed2faa00734b49782c7e84014807e478bc84a
* Result: SUCCEEDED
* [Build Logs](https://bkqkixfb59.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=a64f6281-c563-45d6-b4de-96ab2bc9c6c0%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-local-mode-tests
* Commit ID: 38eed2faa00734b49782c7e84014807e478bc84a
* Result: SUCCEEDED
* [Build Logs](https://8huhkqo601.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=aa042877-b3f6-44cd-8695-90f6c0e01168%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-pr
* Commit ID: 38eed2faa00734b49782c7e84014807e478bc84a
* Result: FAILED
* [Build Logs](https://abkt2mlh3k.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=7d62de80-98e3-4d26-83b3-b5ee02f7cabb%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-slow-tests
* Commit ID: 38eed2faa00734b49782c7e84014807e478bc84a
* Result: SUCCEEDED
* [Build Logs](https://omfrydptb9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=ecfd08b3-2b13-4dd4-9390-326d3e4d7966%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-unit-tests
* Commit ID: 38eed2faa00734b49782c7e84014807e478bc84a
* Result: SUCCEEDED
* [Build Logs](https://fegcnxu6da.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=7fed71cd-2b5c-4b6e-b6d8-51beb2629a1b%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-local-mode-tests
* Commit ID: 4b3f617477d0d246ff4afda766c711587d15e7b3
* Result: SUCCEEDED
* [Build Logs](https://8huhkqo601.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=0c16993d-7374-4608-8bfb-2f51121a542f%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-notebook-tests
* Commit ID: 4b3f617477d0d246ff4afda766c711587d15e7b3
* Result: SUCCEEDED
* [Build Logs](https://bkqkixfb59.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=98d1787c-7915-45d3-a599-43bb945059d9%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-unit-tests
* Commit ID: 4b3f617477d0d246ff4afda766c711587d15e7b3
* Result: SUCCEEDED
* [Build Logs](https://fegcnxu6da.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=8d8de728-3202-44d8-a6d9-e750624bf2ba%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-pr
* Commit ID: 4b3f617477d0d246ff4afda766c711587d15e7b3
* Result: FAILED
* [Build Logs](https://abkt2mlh3k.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=89584d9f-dffd-4bbb-bb5a-bb80798afc8b%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: src/sagemaker/image_uri_config/tasks.json (hunk 7 above)
Comment (knikure) [reply src/sagemaker/image_uri_config/tasks.json:1]: Why this task.json is in this folder? We only have this folder for Image URIs hardcoded configs. Remove it from here.

Comment (makungaj1) [reply src/sagemaker/image_uri_config/tasks.json:1]: Thought you mentioned that this is the only folder that is onboarded with RBA.
I'll move the json file under serve/utils
Fyi @samruds

Comment (samruds) [reply src/sagemaker/image_uri_config/tasks.json:1]: move it to be under serve/schemas (new dir)

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-notebook-tests
* Commit ID: 8f6e6b29add3afe8f7a74f91785fc9628b3a1ad2
* Result: FAILED
* [Build Logs](https://bkqkixfb59.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=cf47ddf2-3957-4c5d-992a-b1b167398ba0%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-local-mode-tests
* Commit ID: 8f6e6b29add3afe8f7a74f91785fc9628b3a1ad2
* Result: SUCCEEDED
* [Build Logs](https://8huhkqo601.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=37d70390-776d-448b-aa7e-817ff741804d%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-slow-tests
* Commit ID: 8f6e6b29add3afe8f7a74f91785fc9628b3a1ad2
* Result: SUCCEEDED
* [Build Logs](https://omfrydptb9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=f20c1f59-b00d-4469-b8eb-931c67bded40%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-pr
* Commit ID: 8f6e6b29add3afe8f7a74f91785fc9628b3a1ad2
* Result: FAILED
* [Build Logs](https://abkt2mlh3k.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=779f4481-f5d8-4447-aa2d-cc1bfbc1ac15%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-unit-tests
* Commit ID: 8f6e6b29add3afe8f7a74f91785fc9628b3a1ad2
* Result: SUCCEEDED
* [Build Logs](https://fegcnxu6da.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=b5ba4860-acbc-448a-90de-af45f130639e%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by mufaddal-rohawala — APPROVED** [review]

File: src/sagemaker/image_uri_config/tasks.json [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+	"fill-mask": {
+        "sample_inputs": {
```
[the remark this answers is not in the document]
Comment (makungaj1) [reply src/sagemaker/image_uri_config/tasks.json]: Moved under serve/schema.

**Review by knikure — COMMENTED** [review]
/bot run notebook-tests, pr

**Review by knikure — APPROVED** [review]

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-notebook-tests
* Commit ID: 8f6e6b29add3afe8f7a74f91785fc9628b3a1ad2
* Result: SUCCEEDED
* [Build Logs](https://bkqkixfb59.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=13ae61b0-6361-42aa-9225-99787978064c%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-pr
* Commit ID: 8f6e6b29add3afe8f7a74f91785fc9628b3a1ad2
* Result: FAILED
* [Build Logs](https://abkt2mlh3k.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=0d051119-e988-4a1a-89fc-fbd30d3c2175%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**